source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc

```diff
@@ -47,11 +47,12 @@ Do not restart the multipathd service. Doing so generates errors in the VDSM log
 
 .Verification steps
 
-Test the new configuration performs as expected on a non-production cluster in a variety of failure scenarios. For example, disable all of the storage connections. Then enable one connection at a time and verify that doing so makes the storage domain reachable.
+. Test the new configuration performs as expected on a non-production cluster in a variety of failure scenarios. For example, disable all of the storage connections.
+. Enable one connection at a time and verify that doing so makes the storage domain reachable.
 
 .Additional resources
 
-*  link:{URL_virt_product_docs}{URL_format}installing_red_hat_virtualization_as_a_self-hosted_engine_using_the_cockpit_web_interface#ref-Recommended_Settings_for_Multipath_conf_SHE_cockpit_deploy[Recommended Settings for Multipath.conf]
+* xref:ref-Recommended_Settings_for_Multipath_conf_{context}[Recommended Settings for Multipath.conf]
```

Comment:
replace {context} with whatever {context} evaluates to.

For example, if the target of the xref is in a location in the document where the value of {context} is `snoopy`, then this xref should be:
` xref:ref-Recommended_Settings_for_Multipath_conf_snoopy`

Reply:
@stoobie See email for questions about this cross reference. Thanks!